src/popup/components/BalanceInfo.vue

```diff
@@ -22,21 +22,22 @@ export default defineComponent({
     AeBalance,
   },
   props: {
-    account: { type: Object, default: null },
+    address: { type: String, default: null },
+    multisigBalance: { type: Number, default: 0 },
   },
   setup(props, { root }) {
     const { balances } = useBalances({ store: root.$store });
-    const { isMultisigDashboard } = useMultisigAccounts({ store: root.$store });
 
     const account = useGetter<IAccount>('account');
     const convertToCurrencyFormatted = useGetter('convertToCurrencyFormatted');
 
-    const currentAccount = computed(() => props.account || account.value);
+    const isMultisigPage = computed(() => !!root.$route.meta?.isMultisigPage);
+    const currentAddress = computed(() => props.address || account.value.address);
 
     const balance = computed(
-      () => (isMultisigDashboard.value
-        ? currentAccount.value.balance
-        : balances.value[currentAccount.value.address])?.toNumber() || 0,
+      () => isMultisigPage.value
+        ? props.multisigBalance
+        : balances.value[currentAddress.value]?.toNumber() || 0,
```

Comment:
In my opinion this component needs to be refactored to receive one prop: `balance` as a number and that's all. Because now it looks really off to have `multisigBalance` and `account` props.